cmd/internal/codegen/parse/crd.go

```diff
@@ -215,28 +215,36 @@ func (b *APIs) parsePrimitiveValidation(t *types.Type, found sets.String, commen
 	return props, buff.String()
 }
 
+type mapTempateArgs struct {
+	Result string
+	SkipMap bool
+}
+
 var mapTemplate = template.Must(template.New("map-template").Parse(
 	`v1beta1.JSONSchemaProps{
     Type:                 "object",
-    AdditionalProperties: &v1beta1.JSONSchemaPropsOrBool{
+    {{if not .SkipMap}}AdditionalProperties: &v1beta1.JSONSchemaPropsOrBool{
         Allows: true,
-        //Schema: &{{.}},
-    },
+        Schema: &{{.Result}},
+    },{{end}}
 }`))
 
 // parseMapValidation returns a JSONSchemaProps object and its serialization in
 // Go that describe the validations for the given map type.
 func (b *APIs) parseMapValidation(t *types.Type, found sets.String, comments []string) (v1beta1.JSONSchemaProps, string) {
-	additionalProps, _ := b.typeToJSONSchemaProps(t.Elem, found, comments)
+	additionalProps, result := b.typeToJSONSchemaProps(t.Elem, found, comments)
 	props := v1beta1.JSONSchemaProps{
 		Type: "object",
-		AdditionalProperties: &v1beta1.JSONSchemaPropsOrBool{
+	}
+    parseOption := b.arguments.CustomArgs.(*ParseOptions)
+	if !parseOption.SkipMap {
+		props.AdditionalProperties = &v1beta1.JSONSchemaPropsOrBool{
 			Allows: true,
-			Schema: &additionalProps},
+			Schema: &additionalProps}
 	}
 
 	buff := &bytes.Buffer{}
-	if err := mapTemplate.Execute(buff, ""); err != nil {
+	if err := mapTemplate.Execute(buff, mapTempateArgs{result, parseOption.SkipMap}); err != nil {
 		log.Fatalf("%v", err)
 	}
 	return props, buff.String()
```

Comment:
Can we write test for this or `create config` command ?

Reply:
Yes, we can. I will update the test.

Reply:
Done